foundation_cms/templates/patterns/pages/core/listing_page.html

```diff
@@ -0,0 +1,4 @@
+{% extends "base.html" %}
+{% block content %}
+    {% include "patterns/components/listing_header.html" with items=pages active_tag=active_tag %}
+{% endblock %}
```

Comment:
Adds a little snippet to print the pages results.

```suggestion
{% block content %}
    {% include "patterns/components/listing_header.html" with items=pages active_tag=active_tag %}
    <div class="listing-links">
        {% for page in pages %}
            <p>
                <a href="{{ page.url }}">{{ page.title }}</a>
            </p>
        {% endfor %}
    </div>
{% endblock %}
```